WordPress/wordpress-playground
# Add mbregex extension to the kitchen-sink bundle

## What is this PR doing?

Adds the `mbregex` extension to the kitchen sink bundle. This also makes it available on the website when the "Load extensions" checkbox is clicked.

Closes https://github.com/WordPress/wordpress-playground/issues/815

## Testing Instructions

1. Apply this PR
2. Open Playground, go to `/phpinfo.php`
3. Look up the mbregex extension and oniguruma
4. Confirm they're both installed
